src/algebra/algebra/basic.lean

```diff
@@ -1516,57 +1401,13 @@ variables {S M N}
 
 namespace submodule
 
-/--
-`V.restrict_scalars R` is the `R`-submodule of the `R`-module given by restriction of scalars,
-corresponding to `V`, an `S`-submodule of the original `S`-module.
--/
-@[simps]
-def restrict_scalars (V : submodule S M) : submodule R M :=
-{ carrier := V.carrier,
-  zero_mem' := V.zero_mem,
-  smul_mem' := λ c m h, V.smul_of_tower_mem c h,
-  add_mem' := λ x y hx hy, V.add_mem hx hy }
-
-@[simp]
-lemma restrict_scalars_mem (V : submodule S M) (m : M) :
-  m ∈ V.restrict_scalars R ↔ m ∈ V :=
-iff.refl _
-
 variables (R S M)
 
-lemma restrict_scalars_injective :
-  function.injective (restrict_scalars R : submodule S M → submodule R M) :=
-λ V₁ V₂ h, ext $ by convert set.ext_iff.1 (set_like.ext'_iff.1 h); refl
-
-@[simp] lemma restrict_scalars_inj {V₁ V₂ : submodule S M} :
-  restrict_scalars R V₁ = restrict_scalars R V₂ ↔ V₁ = V₂ :=
-(restrict_scalars_injective R _ _).eq_iff
-
-@[simp]
-lemma restrict_scalars_bot : restrict_scalars R (⊥ : submodule S M) = ⊥ := rfl
-
-@[simp]
-lemma restrict_scalars_top : restrict_scalars R (⊤ : submodule S M) = ⊤ := rfl
-
 /-- If `S` is an `R`-algebra, then the `R`-module generated by a set `X` is included in the
 `S`-module generated by `X`. -/
 lemma span_le_restrict_scalars (X : set M) : span R (X : set M) ≤ restrict_scalars R (span S X) :=
 submodule.span_le.mpr submodule.subset_span
```

Comment:
Why didn't this one get moved?

Reply:
Because `span` isn't available in the place where I moved the rest of these, and I wasn't feeling super keen to find it a home in `linear_algebra/basic`!

Reply:
(also, keeping it here means that it's near the `eq` version which is probably a good thing)